src/DocumentationBrowserViewExtension/DocumentationBrowserViewModel.cs

```diff
@@ -132,6 +140,8 @@ public bool ShowBrowser
 
         internal UIElement DynamoView { get; set; }
 
+        internal NodeInfo CurrentNodeInfo { get; set; }
```

Comment:
I don't think carrying around this state is ideal - at the minimum it should have some comments...

I'm not sure if it's possible, but it seems simpler to keep track of where this data is used if it's passed in an event or binding (or something similar) to the View.

Just a thought.

Reply:
@mjkkirschner removed the NodeInfo class and I just added a new internal property, check the next commit:
https://github.com/DynamoDS/Dynamo/pull/13703/commits/fdd337c2dd455d53ae51bbd88c17e8a5056ca389